tests/python_client/testcases/test_collection.py

```diff
@@ -3134,16 +3134,20 @@ def test_collection_describe(self):
         description = \
             {'collection_name': c_name, 'auto_id': False, 'num_shards': ct.default_shards_num, 'description': '',
              'fields': [
-                 {'field_id': 100, 'name': 'int64', 'description': '', 'type': 5, 'params': {}, 'is_primary': True},
-                 {'field_id': 101, 'name': 'float', 'description': '', 'type': 10, 'params': {}},
-                 {'field_id': 102, 'name': 'varchar', 'description': '', 'type': 21, 'params': {'max_length': 65535}},
-                 {'field_id': 103, 'name': 'json_field', 'description': '', 'type': 23, 'params': {}},
-                 {'field_id': 104, 'name': 'float_vector', 'description': '', 'type': 101, 'params': {'dim': 128}}
+                 {'field_id': 100, 'name': 'int64', 'description': '', 'type': DataType.INT64, 'params': {},
+                  'is_primary': True},
+                 {'field_id': 101, 'name': 'float', 'description': '', 'type': DataType.FLOAT, 'params': {}},
+                 {'field_id': 102, 'name': 'varchar', 'description': '', 'type': DataType.VARCHAR,
+                  'params': {'max_length': 65535}},
+                 {'field_id': 103, 'name': 'json_field', 'description': '', 'type': DataType.JSON, 'params': {}},
+                 {'field_id': 104, 'name': 'float_vector', 'description': '', 'type': DataType.FLOAT_VECTOR,
+                  'params': {'dim': 128}}
              ],
              'functions': [], 'aliases': [], 'consistency_level': 0, 'properties': {},
              'num_partitions': 1, 'enable_dynamic_field': False}
         res = collection_w.describe()[0]
         del res['collection_id']
+        del res['created_timestamp']
```

Comment:
I suggest we assert created_timestamp and collection_id in the list with the correct value type before delete them